src/components/Hero.jsx

```diff
@@ -119,25 +134,31 @@ const Hero = ({
   buttonText,
 }) => {
   return (
-    <StyledHeroSection>
-      <div className="content-container">
+    <StyledHeroSection className={img ? null : 'no-image'}>
```

Comment:
This is resolving false no matter what. Check the homepage and see that the image is gone. 

It looks like this component was set up to default use the image on the homepage (it's imported in this component rather than the homepage and passed down to the `Hero` component). 

It should be a relatively easy refactor. Just pass the image down on the homepage rather than importing it here. Then change the conditional `or` statement to display this image if there is none.

Reply:
> This is resolving false no matter what. Check the homepage and see that the image is gone.
> 
> It looks like this component was set up to default use the image on the homepage (it's imported in this component rather than the homepage and passed down to the `Hero` component).
> 
> It should be a relatively easy refactor. Just pass the image down on the homepage rather than importing it here. Then change the conditional `or` statement to display this image if there is none.
I think streams were crossed here... I merged with main, then passed the image to HERO on in index page and it seems to be working as intended. so... i think we're good?

Reply:
Looks like for whatever reason the eyebrow text is there. I bet it had something to do with one of the bugs I adjusted so I'll mark this as resolved.